WordPress-VIP-Go/ruleset-test.inc

```diff
@@ -546,7 +546,7 @@ str_replace( 'foo', array( 'bar', 'foo' ), 'foobar' ); // Error.
 
 // WordPressVIPMinimum.Security.Underscorejs
 echo "<script>
- _.templateSettings = { 
+ _.templateSettings = {
```

Comment:
Was this an _intentional_ or _accidental_ trailing whitespace ? I.e. are you changeing what's being tested now or not ?

(same question for the same change in `WordPressVIPMinimum/ruleset-test.inc`)